MekHQ/src/mekhq/gui/dialog/ImageChoiceDialog.java

```diff
@@ -1,9 +1,21 @@
 /*
- * ImageChoiceDialog.java
+ * Copyright (c) 2020  - The MegaMek Team
```

Comment:
```suggestion
 * Copyright (c) 2009, 2016, 2020  - The MegaMek Team
```
Per:
> Created on October 1, 2009, 3:10 PM

and

> $ git log --diff-filter=A -- MekHQ/src/mekhq/gui/dialog/ImageChoiceDialog.java
> commit 24cc46ba7be2de732df3fbe94f959b1df0675497
> Author: Dylan Myers <ralgith@gmail.com>
> Date:   Fri Sep 23 18:57:46 2016 -0400
>
>    + Refactored PortraitChoiceDialog to be ImageChoiceDialog since it's used for more than one thing